app/src/main/java/com/android/sample/model/filter/Filter.kt

```diff
@@ -9,7 +9,13 @@ import com.android.sample.resources.C.Tag.Filter.UNINITIALIZED_BORN_VALUE
 import com.android.sample.resources.C.Tag.FilterPage.TIME_RANGE_MAX
 import com.android.sample.resources.C.Tag.FilterPage.TIME_RANGE_MIN
 
-/** Filter class to store filter types. */
+/**
```

Comment:
The new docs are super clear and well-written! Adding `@param` for each field is a great touch—makes it so much easier to follow. 🙌

Reply:
Thanks